src/utils/handleImportFiles.js

```diff
@@ -0,0 +1,34 @@
+import DocumentPicker from 'react-native-document-picker';
+
+const handleImportFiles = async () => {
+    try {
+        const res = await DocumentPicker.pick({
+            type: [DocumentPicker.types.allFiles], //can change depending on the file type(s) we want to import
+        });
+        // upload the picked file to server
+        const formData = new FormData();
+        formData.append('uploaded-file', {

```

Comment:
```suggestion
        formData.append('file', {
```
should match the name used the backend for multer to work